### ISSUE TITLE
<textarea>s are missing accessible labels

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [replit/jq-console#94]
Hi, thanks for keeping jq-console online all these years!  It's still working well for us at Codecademy. 😄 

Per [WCAG guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html), [form `<input>` elements (such as `<textarea>`s) must have labels](https://dequeuniversity.com/rules/axe/3.2/label). The [input element generated for a console](https://github.com/replit-archive/jq-console/blob/a108eaed6c88586c69f74f59dd3432d3f51153b3/src/jqconsole.coffee#L236) doesn't have an `aria-label` or similar label technique, which is an accessibility issue for our users that are trying to use this with a screenreader.

Would you be open to accepting & releasing a PR that adds `aria-label: "Console text input"` to that element?

```diff
    @$input_source = if @isAndroid then $('<input/>') else $('<textarea/>')
    @$input_source.attr
      wrap: 'off'
+     aria-label: 'Console text input'
      autocapitalize: 'off'
```

**Comment by JoshuaKGoldberg** [comment]
Closing out my old issues in repos that haven't had updates in a few years. Cheers!
